app/jobs/update_all_loans_job.rb

```diff
@@ -3,9 +3,10 @@ def perform(_job_params)
     task = task_for_job(self)
     errors_by_loan = []
     divisions = Division.qb_accessible_divisions
-    loans = divisions.map { |i| i.loans.active }.flatten.compact
     updater = Accounting::Quickbooks::Updater.new
     updater.qb_sync_for_loan_update
+
+    loans = divisions.map { |i| i.loans.changed_since(updater.qb_connection.last_updated_at).active }.flatten.compact
```

Comment:
One thing I missed when I updated this task to do active loans, rather than all loans, is renaming the task to "Update Active Loans." Should we do that so we don't get confused later?